pkgs/development/python-modules/scikitlearn/default.nix

```diff
@@ -61,6 +52,12 @@ buildPythonPackage rec {
 
   meta = with stdenv.lib; {
     description = "A set of python modules for machine learning and data mining";
+    changelog = let
+      major = versions.major version;
+      minor = versions.minor version;
+      dashVer = replaceChars ["."] ["-"] version;
+    in
+      "https://scikit-learn.org/stable/whats_new/v${major}.${minor}.html#version-${dashVer}";
```

Comment:
This is great.  I'm going to update the `nixpkgs-update` bot to drop a link to `meta.description` in the PR description if it exists, btw:

https://github.com/ryantm/nixpkgs-update/pull/200

I spent a lot of time looking for release notes when reviewing PRs, so it'll be good to have it a little more streamlined!